src/Compilers/Core/Portable/WellKnownTypes.cs

```diff
@@ -330,6 +330,9 @@ internal enum WellKnownType
         System_MissingMethodException,
         System_Runtime_CompilerServices_MetadataUpdateOriginalTypeAttribute,
 
+        System_Collections_Generic_Dictionary_KV,
+        System_Collections_Generic_KeyValuePair_KV,
```

Comment:
Consider placing next to:

![image](https://github.com/dotnet/roslyn/assets/4564579/fde88ffc-b33e-42eb-8b36-bff84bb2fb7c)

Reply:
We could although that would require updating the offsets for certain types in `WellKnownMembers`. It seemed simpler to add to the end of this `enum`.